exec/java-exec/src/test/java/org/apache/drill/test/ClusterFixture.java

```diff
@@ -213,7 +204,7 @@ private void configureZk(FixtureBuilder builder) {
     }
   }
 
-  private void createConfig(FixtureBuilder builder) throws Exception {
+  private void createConfig() throws Exception {
```

Comment:
They should be. The only change I made is that I didn't require passing the FixtureBuilder as an arg, because the class is already holding a reference to it.